### PR TITLE
Merge xcode9.1 into master (take 2)

### PR DESCRIPTION
**Pull request by spouliot**
Replace https://github.com/xamarin/xamarin-macios/pull/2953

### CODE REVIEW

File: src/scenekit.cs [hunk 1]
```diff
@@ -2903,14 +3340,17 @@ interface SCNView : SCNSceneRenderer, SCNTechniqueSupport {
 		SCNAntialiasingMode AntialiasingMode { get; set; }
 
 #if XAMCORE_2_0
-		[Watch (4, 0), TV (11, 0), Mac (10, 13, onlyOn64: true), iOS (11, 0)]
+		[Watch (4, 0), TV (11, 0), Mac (10, 13), iOS (11, 0)]
```
Comment (timrisi) [inline src/scenekit.cs]: Why was onlyOn64 removed?

File: src/scenekit.cs [hunk 2]
```diff
@@ -2903,14 +3340,17 @@ interface SCNView : SCNSceneRenderer, SCNTechniqueSupport {
...
 		[Export ("cameraControlConfiguration")]
 		ISCNCameraControlConfiguration CameraControlConfiguration { get; }
 
-		[Watch (4, 0), TV (11, 0), Mac (10, 13, onlyOn64: true), iOS (11, 0)]
```
Comment (timrisi) [inline src/scenekit.cs]: Same question

File: src/scenekit.cs [hunk 3]
```diff
@@ -2903,14 +3340,17 @@ interface SCNView : SCNSceneRenderer, SCNTechniqueSupport {
...
+		[Watch (4, 0), TV (11, 0), Mac (10, 13), iOS (11, 0)]
```
Comment (timrisi) [inline src/scenekit.cs]: onlyOn64?

File: src/scenekit.cs [hunk 4]
```diff
@@ -4363,7 +4833,7 @@ interface SCNBufferStream {
 		unsafe void Length (IntPtr bytes, nuint length);
 	}
 
-	[Watch (4,0), TV (11,0), Mac (10, 13, onlyOn64: true), iOS (11,0)]
```
Comment (timrisi) [inline src/scenekit.cs]: Why not onlyOn64?

File: src/scenekit.cs [hunk 5]
```diff
@@ -4383,7 +4853,7 @@ interface SCNAnimationProtocol {
 
 	interface ISCNAnimationProtocol {}
 
-	[Watch (4,0), TV (11,0), Mac (10, 13, onlyOn64: true), iOS (11,0)]
```
Comment (timrisi) [inline src/scenekit.cs]: Same

File: src/scenekit.cs [hunk 6]
```diff
@@ -4457,7 +4927,7 @@ interface SCNAnimation : SCNAnimationProtocol, NSCopying, NSSecureCoding
 		bool Cumulative { [Bind ("isCumulative")] get; set; }
 	}
 	
-	[Watch (4,0), TV (11,0), Mac (10, 13, onlyOn64: true), iOS (11,0)]
```
Comment (timrisi) [inline src/scenekit.cs]: Same

File: src/scenekit.cs (hunk 1 above)
Comment (spouliot) [reply src/scenekit.cs]: @timrisi it breaks the build for classic. I'm not sure why/when those were added in master and why I asked your review ;-)

**Comment by spouliot** [comment]
Issues raised by @timrisi needs to be addressed in 15.5 first (and then go back to master) in order to avoid future breaking changes.

**Pull request merged**